pkg/internal/cluster/create/create.go

```diff
@@ -190,12 +191,20 @@ func printUsage(name string) {
 			name,
 		)
 	} else {
+		// Detect shell and usage build prompt appropriately.
+		setKubeConfigPrompt := ""
+		switch shell := env.GetLinuxShell(); shell {
```

Comment:
shouldn't the function be called `UnixShell`, as this branch is `!windows`?
can it cover custom OSX shells too?

Reply:
also noting that fish appears to be on macOS